mantidproject/mantid
# Indirect I(Q,t)Fit - Update Plot Options

A new plot option has been added to I(Q,t)Fit so that Background is plotted. By doing this, the bug causing `Beta` to be listed twice and also the bug which causes the selected plot option to plot the wrong parameter should be resolved.

**To test:**
Input Files:
[irs26176_graphite002_iqt.zip](https://github.com/mantidproject/mantid/files/732557/irs26176_graphite002_iqt.zip)
Interfaces>Indirect>DataAnalysis>I(Q,t)Fit
Load workspace
Change `EndX` to be `~0.18`
Press `Run`
Check plot options menu. For `Exponential` fit types it should be:
```
Background
Intensity
Tau
All
```
For `Stretched Exponential` it should be as above with an addition `Beta` option.
Check that:
Background plots A0
Intensity plots Height
Tau plots Lifetime
Beta plots Stretching
All plots all of the above on separate graphs.


Fixes #18570.

*Release Notes have been updated*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
